guides/user/triggers/webhooks/index.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: single
+title:  "Triggering on Webhooks"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+In order to programatically trigger pipelines you can send a `POST` call to
+Spinnaker against a preconfigured endpoint. This can be used to trigger
+pipelines when a CI job finishes, from a command line, or from a third-party
+system.
+
+> __☞ Note__: There is not a 1:1 mapping between webhooks and pipeline executions.
+> It's possible to configure multiple pipelines to trigger off of a single
+> webhook.
+
+## Adding a Webhook Trigger to a Pipeline
+
+Assuming you have created a pipeline, under __Configuration__, select __Add
+Trigger__ and make its type selector __Webhook__.
+
+To assign an endpoint that must be hit, you can provide a value to the
+__Source__ field as shown here:
+
+{%
+  include
+  figure
+  image_path="./basic-webhook.png"
+%}
+
+Notice that in the above image below the __Type__ dropdown, the webhook
+configuration points out that we can hit
+`http://localhost:8084/webhooks/webhook/demo` to trigger the pipeline. The
+endpoint depends on how you've configured your Spinnaker endpoints -- if you're
+running on a different endpoint, e.g. `https://api.spinnaker-prod.net`, that'll
+be shown instead.
+
+Keeping track of that endpoint as `$ENDPOINT` (it will depend on where
+Spinnaker is installed), save that pipeline, and run:
+
+```bash
+curl $ENDPOINT -X POST -H "content-type: application/json" -d "{ }"
+```
+
+### Payload Constraints
+
+If you want to ensure that a webhook only triggers when a certain payload
+arrives, you can provide __Payload Constraints__. These are key/value pairs
```

Comment:
"...you can provide __Payload Contraints__ in the trigger." 

I think it's pretty clear anyway, but it doesn't hurt, since we're configuring the webhook on one end and the trigger on the other, to be clear about where we do each thing.